pkg/controller/cainjector/indexers.go

```diff
@@ -212,3 +214,33 @@ func injectableCAFromSecretIndexer(rawObj client.Object) []string {
 
 	return []string{secretNameRaw}
 }
+
+// isInjectable returns predicates that determine whether an object is a
+// cainjector injectable by looking at whether it has one of the three
+// annotations used to mark injectables.
+func isInjectable() predicate.Funcs {
+	f := func(o client.Object) bool {
+		annots := o.GetAnnotations()
+		if _, ok := annots[cmapi.WantInjectAPIServerCAAnnotation]; ok {
+			return true
+		}
+		if _, ok := annots[cmapi.WantInjectAnnotation]; ok {
+			return true
+		}
+		if _, ok := annots[cmapi.WantInjectFromSecretAnnotation]; ok {
+			return true
+		}
+		return false
+	}
```

Comment:
Suggest changing this nested function to an ordinary function. It don't think it needs any variables from the parent function.